tests/test_010_wells.py

```diff
@@ -50,6 +50,15 @@ def test_wel_from_df():
     nlmod.gwf.wells.wel_from_df(wells, gwf)
 
 
+def test_wel_from_df_no_multiplier():
+    wells = pd.DataFrame(columns=["x", "y", "top", "botm", "Q"], index=range(2))
+    wells.loc[0] = 100, -50, -5, -10, -100.0
+    wells.loc[1] = 200, 150, -20, -30, -300.0
+
+    sim, gwf = get_sim_and_gwf()
+    nlmod.gwf.wells.wel_from_df(wells, gwf, auxmultname=None)
```

Comment:
What's the difference between the auxmultname=None and setting it to "multiplier"? In both cases a multiplier is applied, but either via AUX variables or just directly in the perioddata?

Maybe a docstring in wel_from_df()  would help clear that up

Reply:
Yes, you are right. I added a docstring.